### PR TITLE
ceph-iscsi-config-flake8: initial version

### PR DESCRIPTION
**Pull request by dillaman**
Signed-off-by: Jason Dillaman <dillaman@redhat.com>

### CODE REVIEW

File: ceph-iscsi-config-flake8/build/build [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   fi
+
+}
+
+function check(){
+    local file
+    while read -r filename; do
+        pushd "$(dirname "$filename")"
+        file=$(basename "$filename")
+        sudo docker run --rm -v "$workspace"/ceph-iscsi-config/tox.ini:/tox.ini -v "$(pwd)"/"$file":/"$file" eeacms/flake8 /"$file"
```
Comment (alfredodeza) [inline ceph-iscsi-config-flake8/build/build]: why is docker required to run flake8 here?

Comment (dillaman) [reply ceph-iscsi-config-flake8/build/build]: No clue -- I copied from another flake8 checker

Comment (dillaman) [reply ceph-iscsi-config-flake8/build/build]: ... and I'd imagine this will eventually need docker for future integration tests

Comment (ktdreyer) [reply ceph-iscsi-config-flake8/build/build]: @dillaman https://stickler-ci.com/ is an alternative for running flake8 on Python projects. I've used it for a couple projects on Github, and it was really easy to set up.

**Pull request merged**